qutip/tests/solver/test_correlation.py

```diff
@@ -127,6 +126,7 @@ def _2ls_g2_0(H, c_ops):
     sp = qutip.sigmap()
     start = qutip.basis(2, 0)
     times = _2ls_times
+    H = qutip.QobjEvo(H, args=_2ls_args, tlist=times)
```

Comment:
Just a minor question here. Why is it necessary to explicitly make H a QobjEvo? I am worried that if this is necessary there might have been a breaking change.

Reply:
To test array based coefficient, we need to pass a `tlist`. `correlation` has `tlist` and `taulist` which would case confusion as to which should be used. 
In v4, array as coefficient where accepted, but it uses `tlist` in the first call and `taulist` in the following, not user how it handled the shift in the taus.